.github/workflows/push-oci-pr.yml

```diff
@@ -25,40 +25,49 @@ on:
   
 jobs:
   push:
-    name: push
+    name: artifact-check-then-push
     if: |
       github.event.workflow_run.conclusion == 'success' &&
       github.event.workflow_run.event == 'pull_request'
     runs-on: ubuntu-24.04
-    permissions:
-      contents: read
-      packages: write
     steps:
-      - name: Download mapt assets
+      - name: Try to Download Artifact
```

Comment:
:) Why try?....just out of curiosity why change the assertion.... (download) for a supposition (try to downlowad) ?

Reply:
Ahh b/c there is no way to get the job name to filter it?